node.gyp

```diff
@@ -119,6 +119,7 @@
       'lib/internal/readline.js',
       'lib/internal/repl.js',
       'lib/internal/socket_list.js',
+      'lib/internal/tls.js',
```

Comment:
Is there a reason this needs to be here twice? (It seems to already exist on line 124.)

Reply:
No, I think that is some sort of merge conflict.